AK Apolo/app/src/main/java/ayds/apolo/songinfo/moredetails/fulllogic/ArtistDataBase.kt

```diff
@@ -0,0 +1,75 @@
+package ayds.apolo.songinfo.moredetails.fulllogic
+
+import android.database.sqlite.SQLiteOpenHelper
+import android.database.sqlite.SQLiteDatabase
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+
+internal const val ARTIST = "artist"
+internal const val ARTISTS = "artists"
+internal const val INFO = "info"
+internal const val SOURCE = "source"
+internal const val ID = "id"
+internal const val VERSION = 1
+internal const val SELECTION = "artist = ?"
+internal const val SORT_ORDER = "artist DESC"
+internal const val DATABASE_NAME = "dictionary.db"
+
+class ArtistDataBase(context: Context?): SQLiteOpenHelper(context, DATABASE_NAME, null, VERSION) {
+    override fun onCreate(db: SQLiteDatabase) {
+        db.execSQL(
+            "create table $ARTISTS ($ID INTEGER PRIMARY KEY AUTOINCREMENT, $ARTIST string, $INFO string, $SOURCE integer)"
+        )
+    }
+
+    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}
+
+    fun saveArtist(artist: String?, info: String?) {
+        val source = 1
+        val values = ContentValues().apply {
+            put(ARTIST, artist)
+            put(INFO, info)
+            put(SOURCE, source)
+        }
+
+        writableDatabase.insert(ARTISTS, null, values)
+    }
+
+    fun getInfo(artist: String): String? {
+        val cursor = getCursor(arrayOf(artist))
+
+        return cursor.getInfoFromCursor()
+    }
+
+    private fun getCursor( selectionArgs: Array<String>): Cursor {
+        val projection = arrayOf(
+            ID,
+            ARTIST,
+            INFO
+        )
+        return writableDatabase.query(
+            ARTISTS,
+            projection,
+            SELECTION,
+            selectionArgs,
+            null,
+            null,
+            SORT_ORDER
+        )
+    }
+
+    private fun Cursor.getInfoFromCursor():String?{
+        var info:String? = null
```

Comment:
Mala separación de simbolos
```suggestion
        var info: String? = null
```